translations/en.json

```diff
@@ -1,4 +1,36 @@
 {
+  "Active Medications": "Active Medications",
```

Comment:
We should remove the babel plugin for automatic translation generation.

For the keys we should prefer JS identifiers.